source/nodejs/adaptivecards-designer-app/dist/containers/cortana-container.css

```diff
@@ -239,11 +239,14 @@ a.ac-anchor:visited:active {
     height: 72px;
 }
 
+.ac-input.ac-textInput, .ac-input.ac-numberInput, .ac-input.ac-dateInput, .ac-input.ac-timeInput {
+    height: 31px;
```

Comment:
Do we not need this height value on the `multichoiceInput` class?

Reply:
Sooooo, once again, this file seems auto-generated, yet committed to the repo...

Reply:
Marking this as resolved as this is a pre-existing idiosyncrasy